src/Commands/Load.php

```diff
@@ -30,7 +30,13 @@ public function handle()
             return;
         }
 
-        $name = $this->argument('name') ?: $this->askForSnapshotName();
+        $use_latest_snapshot = $this->option('latest') ?: false;
+
+        if ($use_latest_snapshot) {
+            $name = $snapShots->last()->name;
+        } else {
```

Comment:
Avoid using `else`, use ternaries instead.

Reply:
I was conflicted here since the else has a ternary operator already